prod-stack.py

```diff
@@ -754,6 +760,27 @@
                 )
             ),
             ('AWS_REGION', Sub('${AWS::Region}')),
+            ('GAME', 'KSP')
+        ],
+        'volumes': [
+            ('ckan_cache', '/home/netkan/ckan_cache')
+        ]
+    },
+    {
+        'name': 'InflatorKsp2',
+        'image': 'kspckan/inflator',
+        'memory': '256',
```

Comment:
Looks like the original Inflator had `156` before #224, we could probably start at least that low here to save memory given that both repos are currently <200KiB and the downloads are at least that tiny with no part or planet packs yet:

```
$ du -sh KSP2*
153K    KSP2-CKAN-meta
130K    KSP2-NetKAN
```

Reply:
True, though it's the max memory they're allowed to use rather then allocated. There is just shy of 900MB headroom for it, so we should be ok.

Reply:
Works for me!